wren-ui/Dockerfile

```diff
@@ -1,14 +1,28 @@
-FROM node:18-alpine AS base
+FROM node:18-bookworm-slim AS base
+
+# Install required packages
+RUN apt-get update && apt-get install -y curl && rm -rf /var/lib/apt/lists/*
+
+# Next.js collects completely anonymous telemetry data about general usage.
+# Learn more here: https://nextjs.org/telemetry
+# Uncomment the following line in case you want to disable telemetry during the build.
+ENV NEXT_TELEMETRY_DISABLED 1
+ENV NODE_ENV production
```

Comment:
I think we could remove this line in base stage image. Setting NODE_ENV to prod will have a larger impact than we thought. If you want yarn installation to skip dev dependencies, simply do this in `deps` stage, or add to `yarn install` (`NODE_ENV=production yarn install --immutable`) would be better, IMO.

seeking advice from @onlyjackfrost  as well.

Reply:
I'm thinking you could just keep `apt-get update ...` commands in base stage image. Others commands that are related to `yarn` installation could be moved to `deps` stage.

Reply:
ohh, we did have `ENV NODE_ENV production` at runner stage before, I guess it's fine. Let's keep it the same way.